driver/src/test/java/org/kaazing/k3po/driver/BehaviorIT.java

```diff
@@ -42,9 +42,23 @@
 
     @Test
     @TestSpecification({
-        "many.expressions" })
+        "many.juel.expressions" })
```

Comment:
"many.el.expressions" ?  (JUEL is implementation of EL, syntax in scripts is EL)